aicsmlsegment/model_utils.py

```diff
@@ -3,10 +3,11 @@
 from monai.inferers import sliding_window_inference
 
 
-def apply_on_image(model, input_img, args):
+def apply_on_image(model, input_img, args, squeeze, to_numpy):
     """
     Perform inference on an input img through a model with or without runtime augmentation.
     If runtime augmentation is selected, perform inference on flipped images and average results.
+    returns: 5d np.array NCZYX order
```

Comment:
can you add docstring for Parameters to give more details about each parameter here?